lms/djangoapps/grades/tests/test_course_data.py

```diff
@@ -0,0 +1,56 @@
+"""
+Tests for CourseData utility class.
+"""
+from lms.djangoapps.course_blocks.api import get_course_blocks
+from mock import patch
+from student.tests.factories import UserFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+from ..new.course_data import CourseData
+
+
+class CourseDataTest(ModuleStoreTestCase):
+    """
+    Simple tests to ensure CourseData works as advertised.
+    """
+
+    def setUp(self):
+        super(CourseDataTest, self).setUp()
+        self.factory_course = CourseFactory.create()
```

Comment:
Honestly, it was a mass-rename that occurred when I was writing the test to differentiate from all the other courses. I'll revert back.